### ISSUE TITLE
Bug: Property 'vm' does not exist on type 'WrapperLike'

### ISSUE THREAD

**Issue by MikeEtlingerRM** [vuejs/test-utils#1498]
Hey `test-utils` team!
In an effort to upgrade Vite, I had to upgrade Vue and subsequently `test-utils`. Unfortunately, this broke my tests that rely on `wrapper.vm` where now a new type called `WrapperLike` is returned in place of `vm`. 

**UPDATE**: Was able to get unblocked by not upgrading `test-utils`. Leaving this here for tracking purposes.

**Describe the bug**
In upgrading from `2.0.0-rc.16 -> 2.0.0-rc.21`(breaking at `2.0.0-rc.18` I think), when using `findComponent` and `getComponent`, I get the following error messages when trying to access `vm` on the return values.

> Property 'vm' does not exist on type 'Omit<WrapperLike, "exists">'

I looked through the release changes and nothing jumped out as the culprit, so I'm not exactly sure what happened when upgrading but I thought it would be worth calling out as a potential bug people are running into.

**To Reproduce**
I will try to get you a proper repro, but I wanted to flag this in case anyone has a quick fix.

Here's the typical scenario where this is breaking:

```ts
it("should get the percentage", () => {
    // Arrange

    const wrapper = shallowMount(MyComponent, {
      myProp: 1,
    });

    // Act
    const percentageComponent = wrapper.findComponent(
      '[test-id="my-testing-id"'
    );

    // Assert
    expect((percentageComponent.vm as any).value).toEqual("33%");
  });
// =>  error TS2339: Property 'vm' does not exist on type 'WrapperLike'.
```

**Expected behavior**
In version `2.0.0-rc.16`, this was working as expected, without errors. I appreciate your help while I try to repro this or get around this somehow 🙏 

**Related information:**
- `@vue/test-utils` version: `2.0.0-rc.21`
- `Vue` version: `3.2.33`
- `node` version: `14.15.4`
- `npm` (or `yarn`) version: `6.14.10`

**Additional context**

**Comment by MikeEtlingerRM** [comment]
Update: I was able to upgrade Vite without updating `test-utils`. Still don't know why I got the error but feel free to close this if you want.

**Comment by freakzlike** [comment]
I recently added some docs for the `WrapperLike` when using `findComponent`/`getComponent`

See the warning box: `WrapperLike type when using CSS selector`
https://test-utils.vuejs.org/guide/advanced/component-instance.html#usage-with-getcomponent-and-findcomponent

**Comment by MikeEtlingerRM** [comment]
@freakzlike Ohhh, nice! I read the docs and saw that `get/findComponent` should return a `VueWrapper` but I must not have scrolled down all the way.

This is super helpful, thanks!

**Comment by freakzlike** [comment]
You're welcome. I'll close this. Feel free to open a new issue in case you find a bug.
